kaankoken/versioning-tool
# Fixing base pipeline structure

- Add `test_coverage` html added to project
- Fix permission of `badge_workflow` is corrected
- Update `package` version update.
- Fix linter, static check & vet correction for the pipeline
- Add script (`run_command`) for running similar purpose commands
